README.md

```diff
@@ -60,9 +60,17 @@ OPTIONS:
         --texture2 <texture2>    Path to 2D RGBA texture for iChannel2
         --texture3 <texture3>    Path to 2D RGBA texture for iChannel3
     -W, --width <width>          Sets window width [default: 600]
+        --wrap0 <wrap0>          Wrap mode for iChannel0 [default: clamp]  [possible values: clamp, tile, mirror,
+                                 border]
+        --wrap1 <wrap1>          Wrap mode for iChannel1 [default: clamp]  [possible values: clamp, tile, mirror,
```

Comment:
@fmenozzi as you can see formatting from the `--help` text has awkward line wrapping.  Want me to leave it or move the possible values lines down?

Reply:
Hmm yeah I figured it would be a bit smarter about the wrapping. In that case, it would probably look nicer to have the "[possible values: ..." line moved to the next line, flush with "Wrap mode for ...".

Reply:
Updated :+1:

Reply:
Thanks! Looks much better now